sdcm/remote/base.py

```diff
@@ -111,8 +111,15 @@ def sudo(self,
              new_session: bool = False,
              log_file: Optional[str] = None,
              retry: int = 1,
-             watchers: Optional[List[StreamWatcher]] = None) -> Result:
-        return self.run(cmd=cmd if self.user == "root" else f"sudo {cmd}",
+             watchers: Optional[List[StreamWatcher]] = None,
+             user: Optional[str] = 'root'
+             ) -> Result:
```

Comment:
Looks pretty ugly for my taste.

Reply:
agree

Reply:
`.... user: Optional[str] = 'root') -> Result:` is better style, let's use it